services/pushover.py

```diff
@@ -14,19 +22,29 @@
 
 class PushoverError(Exception): pass
 
-def pushover(**kwargs):
-    assert 'message' in kwargs
+def pushover(message, user, token, params, filepayload=None):
+
+    if token is None:
+        params['token'] = os.environ['PUSHOVER_TOKEN']
+    else:
+        params['token'] = token
+    #params['token'] = token
 
-    if not 'token' in kwargs:
-        kwargs['token'] = os.environ['PUSHOVER_TOKEN']
-    if not 'user' in kwargs:
-        kwargs['user'] = os.environ['PUSHOVER_USER']
+    #if user is None:
```

Comment:
Why is this commented out? Should be included to keep same functionality as before?